libs/wingsdk/src/target-sim/service.inflight.ts

```diff
@@ -56,7 +56,7 @@ export class Service implements IServiceClient, ISimulatorResourceInstance {
       timestamp: new Date().toISOString(),
     });
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    fnClient.invoke("");
+    await fnClient.invoke("");
```

Comment:
Hmmm, I feel like we just added this because there was an issue with cloud.Service blocking. Let me see if I can find the issue.

Reply:
https://github.com/winglang/wing/issues/4086

Reply:
The use case of blocking the thread in onStart doesn't make sense to me.

Reply:
Removed the "loop" test and added documentation to explain that `onStart()` should not block. Let's see how this works.